lib/identity/client.js

```diff
@@ -233,14 +232,13 @@ class Client extends PartialClient {
   /**
    * downloadFile
    *
-   * @param {Object} recordId
+   * @param {Object} record the record that contains the file info or the recordId
    *
    * @return {string}
    */
-  async downloadFile(recordId) {
-    const file = await this.storage.getFile(recordId)
-    const url = await fileAsUrl(file)
-    return url
+  async downloadFile(record) {
```

Comment:
nit: Since this is a direct client method, I'm wondering if this method is descriptive enough. Perhaps we just use `getFile` so it's an alias to the storage method, just allowing easier access.